lib/power_action_utils.pm

```diff
@@ -277,12 +277,23 @@ sub power_action {
         }
         elsif ($action eq 'poweroff') {
             if (is_backend_s390x) {
-                record_info('poo#114439', 'Temporary workaround, because shutdown module is marked as failed on s390x backend when shutting down from GUI.');
-                select_console 'root-console';
-                enter_cmd "$action";
-            }
-            else {
-                poweroff_x11;
+                if (check_var("DESKTOP", "gnome")) {
+                    send_key "ctrl-alt-delete";
+                    assert_screen 'logoutdialog', 15;
+                    assert_and_click 'gnome-shell_shutdown_btn';
+                    if (get_var("SHUTDOWN_NEEDS_AUTH")) {
+                        assert_screen 'shutdown-auth';
+                        type_password;
+                        # we need to kill all open ssh connections before the system shuts down
+                        prepare_system_shutdown;
+                        send_key "ret";
+                    }
+                }
```

Comment:
How about
```
if (check_var("DESKTOP", "gnome")) {
    poweroff_x11;
}
```
?

Reply:
tried already this is not working, zVM is different then zKVM, thanks for your input. Now I have changed this and will update with verification run :)